haystack/retriever/base.py

```diff
@@ -170,9 +175,13 @@ def eval(
 
     def run(self, pipeline_type: str, **kwargs): # type: ignore
         if pipeline_type == "Query":
-            output, stream = self.run_query(**kwargs)
+            self.query_count += 1
+            run_query = self.timing(self.run_query, "query_time")
```

Comment:
For readability: run_query_timed instead of run_query?

Same below.

Reply:
agreed